### ISSUE TITLE
Can i ask you some questions?

### ISSUE THREAD

**Issue by lfwin** [nagadomi/kaggle-cifar10-torch7#7]
Do you use momentum + SGD to optimize this CNN?
How to you set your learning rate?
I use theano to realize the same CNN structure like the one you gave, but only get a very bad validation error

**Comment by nagadomi** [comment]
> Do you use momentum + SGD to optimize this CNN?

Yes.
https://github.com/nagadomi/kaggle-cifar10-torch7/blob/cuda-convnet2/train.lua#L14
(actual learning rate = 1.0/xBatchSize \* learningRate)
I don't know about theano.
